src/common.jl

```diff
@@ -319,23 +359,30 @@ const XPRS_GLOBALFILESIZE = 1252
 const XPRS_GLOBALFILEUSAGE = 1253
 const XPRS_INDICATORS = 1254
 const XPRS_ORIGINALINDICATORS = 1255
+const XPRS_CORESPERCPUDETECTED = 1258
+const XPRS_CPUSDETECTED = 1259
 const XPRS_CORESDETECTED = 1260
+const XPRS_PHYSICALCORESDETECTED = 1261
+const XPRS_PHYSICALCORESPERCPUDETECTED = 1262
 const XPRS_BARSING = 1281
 const XPRS_BARSINGR = 1282
 const XPRS_PRESOLVEINDEX = 1284
 const XPRS_CONES = 1307
 const XPRS_CONEELEMS = 1308
 const XPRS_BARITER = 5001
-const XPRS_BARAASIZE = 5002
-const XPRS_BARLSIZE = 5003
```

Comment:
maybe leave the older one and add comment sayn its deprecated

Reply:
The values for these aren't even exported anymore. I don't think we should make them available.